src/api.js

```diff
@@ -28,4 +28,33 @@ export const patchArticleVotes = (article_id, vote) => {
     return ncNews.patch(`articles/${article_id}`, body).then((res) => {
         return res.data.article
     })
-}
\ No newline at end of file
+}
+
+export const getUser = (username) => {
+    return ncNews.get("/users").then((res) => {
+        const user = res.data.users.filter((user) => { return user.username === username})
+        return !user.length ? "not found": user[0] 
+    })
+}
+
+export const postComment = (user, comment_body, article_id) => {
+    const body = {
+        username: user,
+        body: comment_body
+    }
+    return ncNews.post(`/articles/${article_id}/comments`, body).then((res) => {
+        return res.data.comment
+    })
+}
+
+export const getUsers = () => {
+    return ncNews.get("/users").then((res) => {
+        return res.data.users
+    })
+}
+
+export const getArticlesByTopic = (topic) => {
+    return ncNews.get("/articles").then((res) => {
+        return res.data.articles.filter((article) => article.topic === topic)
+    })
+}
```

Comment:
We don't want to filter our topics like this.

We want to query our api (/articles?topic=apples for example) to get the filtered results.

Rewrite your getArticles function so that we can use those queries (look up using params with axios)